tools/accuracy_checker/accuracy_checker/postprocessor/shift.py

```diff
@@ -55,3 +55,34 @@ def process_image(self, annotation, prediction):
             prediction_.mask = update_mask.astype(np.int16)
 
         return annotation, prediction
+
+
+class ShiftLabels(PostprocessorWithSpecificTargets):
+    """
+    Shift predicted detection labels or annotation detection labels.
+    """
+
+    __provider__ = 'shift_labels'
+
+    prediction_types = (DetectionPrediction, )
+    annotation_types = (DetectionAnnotation, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'offset': NumberField(value_type=int, optional=True, default=0, description="Value for shift.")
+        })
+        return parameters
+
+    def configure(self):
+        self.offset = self.get_value_from_config('offset')
+
+    def process_image(self, annotation, prediction):
+        for prediction_ in prediction:
+            prediction_.labels += self.offset
+
+        for annotation_ in annotation:
+            annotation_.labels += self.offset
+
+        return annotation, prediction
```

Comment:
@jkamelin could you please describe why these changes need?
Is it impossible to align labels during annotation conversion?

Reply:
These changes were made for uniformity, for this specific model, shift is only needed for predictions, in this case code for annotations can be removed. Do you suggest to perform align not in postprocessor, but during annotation conversion?

Reply:
no, i think for now it is ok. If we change logic in converter, it will be not backward compatible